index.html

```diff
@@ -127,6 +94,6 @@ <h2 class="achievement-title">Achievement unlocked!</h2>
         <button class="btn btn-popup" id="btn-popup">Play on!</button>
       </div>
     </div>
```

Comment:
The pop ups should be rendered in JS as well, so let's not forget to move it out of the HTML